### PR TITLE
Rotated stream

### PR DESCRIPTION
**Pull request by Eddasol**

### CODE REVIEW

File: backend/api/Database/Models/VideoStream.cs [hunk 1]
```diff
@@ -23,5 +23,8 @@ public class VideoStream
         [MaxLength(64)]
         [Required]
         public string Type { get; set; }
+
+        [MaxLength(64)]
+        public bool ShouldRotate { get; set; }
```
Comment (oysand) [inline backend/api/Database/Models/VideoStream.cs:28]: Should we specify the degree to rotate here?

Comment (oysand) [reply backend/api/Database/Models/VideoStream.cs:28]: Should it be required?

Comment (Eddasol) [reply backend/api/Database/Models/VideoStream.cs:28]: We could, but currently we only support 270 deg rotation. Would renaming to ShouldRotate270 be better?

Comment (Eddasol) [reply backend/api/Database/Models/VideoStream.cs:28]: I don't think it should be required as this is only relevant for some robots

File: frontend/src/components/Pages/MissionPage/VideoStream/VideoStreamWindow.tsx [hunk 2]
```diff
@@ -30,33 +58,97 @@ interface VideoStreamWindowProps {
...
     }
 
     return (
         <VideoCard variant="default" style={{ boxShadow: tokens.elevation.raised }}>
-            <StyledVideoSection>{videoPlayer}</StyledVideoSection>
+            <div onDoubleClick={turnOnFullScreen}>{videoPlayer}</div>
             <Typography variant="h5">{videoStream.name}</Typography>
         </VideoCard>
     )
 }
```
Comment (Eddasol) [inline frontend/src/components/Pages/MissionPage/VideoStream/VideoStreamWindow.tsx:114]: Split the file here

File: frontend/src/components/Pages/MissionPage/VideoStream/VideoStreamWindow.tsx [hunk 3]
```diff
@@ -30,33 +58,97 @@ interface VideoStreamWindowProps {
...
+        <FullscreenButton color="secondary" onClick={turnOnFullScreen}>
+            <Icon name="fullscreen" size={32} />
+        </FullscreenButton>
+    )
+
     if (IsValidOvenPlayerType({ videoStream })) {
-        videoPlayer = <VideoPlayerOvenPlayer videoStream={videoStream} />
+        if (videoStream.shouldRotate) {
+            videoPlayer = (
+                <StyledVideoSection style={{ width: '10rem' }}>
```
Comment (oysand) [inline frontend/src/components/Pages/MissionPage/VideoStream/VideoStreamWindow.tsx:82]: Create a new styled componenent from StyledVideoSection

File: backend/api/Database/Models/VideoStream.cs (hunk 1 above)
Comment (oysand) [reply backend/api/Database/Models/VideoStream.cs:28]: Agreed, should also state CW or CCW

**Review by GodVenn — APPROVED** [review]
LGTM

**Pull request merged**